code/modules/mob/living/carbon/human/species_types/ethereal.dm

```diff
@@ -8,7 +8,7 @@
 	mutantlungs = /obj/item/organ/lungs/ethereal
 	mutantstomach = /obj/item/organ/stomach/cell/ethereal
 	mutantheart = /obj/item/organ/heart/ethereal
-	exotic_blood = /datum/reagent/consumable/liquidelectricity //Liquid Electricity. fuck you think of something better gamer
+	exotic_bloodtype = "E"
```

Comment:
at no point do you actually define an "E" /datum/blood_type

Reply:
resolved